src/views/RecIM/components/Header/index.js

```diff
@@ -24,22 +21,27 @@ const RecIMBreadcrumb = ({ link, children }) => {
   return <Typography color="text.primary">{children}</Typography>;
 };
 
-const Header = ({ page, expandable = false, match, team, activity, home, admin }) => {
+const Header = ({
+  page,
+  expandable = false,
+  match,
+  team,
+  activity,
+  home,
+  admin,
+  setOpenHeaderForm,
+  setSelectedMatchScores, //will be removed when we have a single button on the match edit
+}) => {
   const { profile } = useUser();
   const [user, setUser] = useState();
-  const [openActivityForm, setOpenActivityForm] = useState(false);
-  const [openTeamForm, setOpenTeamForm] = useState(false);
   const [isCaptain, setIsCaptain] = useState(false);
-  const [team0Score, setTeam0Score] = useState(0);
-  const [team1Score, setTeam1Score] = useState(0);
-  const [openEditMatchStatsForm, setOpenEditMatchStatsForm] = useState(false);
-  const [selectedScores, setSelectedScores] = useState();
```

Comment:
Having the form logic within the header makes this component **far** too bloated. Furthermore, it prevents refresh on save since the fetch is happening on the actual ____ page